statsmodels/datasets/utils.py

```diff
@@ -338,3 +286,61 @@ def check_internet(url=None):
     except URLError as err:
         return False
     return True
+
+
+def strip_column_names(df):
+    """
+    Remove leading and trailing single quotes
+
+    Parameters
+    ----------
+    df : DataFrame
+        DataFrame to process
+
+    Returns
+    -------
+    df : DataFrame
+        Dataframe with stripped column names
+
+    Notes
+    -----
+    In-place modification
+    """
+    columns = []
+    for c in df:
+        if c.startswith('\'') and c.endswith('\''):
+            c = c[1:-1]
+        elif c.startswith('\''):
+            c = c[1:]
+        elif c.endswith('\''):
+            c = c[:-1]
+        columns.append(c)
+    df.columns = columns
+    return df
+
+
+def load_csv(base_file, csv_name, sep=',', convert_float=False):
+    """Standard simple csv loader"""
+    filepath = dirname(abspath(base_file))
+    filename = join(filepath,csv_name)
+    engine = 'python' if sep != ',' else 'c'
+    float_precision = {}
+    if engine == 'c':
+        float_precision = {'float_precision': True}
```

Comment:
What does "True" select?

according to https://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_csv.html

```
float_precision : string, default None

Specifies which converter the C engine should use for floating-point values. The options are None for the ordinary converter, high for the high-precision converter, and round_trip for the round-trip converter.
```

Reply:
Thanks - wrong reading of doc string.